### PR TITLE
Modifying private to internal

### PR DESCRIPTION
**Pull request by Dyonishw**
Modifying the comment stating default visibility for state variables from "private" to "internal"

### CODE REVIEW

**Comment by Dyonishw** [comment]
http://solidity.readthedocs.io/en/develop/contracts.html#visibility-and-getters

**Comment by maurelian** [comment]
> internal:
> Those functions and state variables can only be accessed internally (i.e. from within the current contract or contracts deriving from it), without using this.

Looks right to me, and is a more intuitive behavior too.

File: docs/recommendations.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ Explicitly label the visibility of functions and state variables. Functions can
 
 ```sol
 // bad
-uint x; // the default is private for state variables, but it should be made explicit
+uint x; // the default is internal for state variables, but it should be made explicit
```
Comment (maurelian) [inline docs/recommendations.md]: Great, thanks for reading carefully!

Comment (Dyonishw) [reply docs/recommendations.md]: I think this repo is a great source of info and I'll be glad to help if I find anything else. Thank you

**Pull request merged**